var/spack/repos/builtin/packages/wireshark/package.py

```diff
@@ -65,7 +69,11 @@ def cmake_args(self):
         args = ['-DENEABLE_CARES=ON',
                 '-DENABLE_GNUTLS=ON',
                 '-DENABLE_LUA=ON',
-                '-DENABLE_MAXMINDDB=ON']
+                '-DENABLE_MAXMINDDB=ON',
+                '-DYACC_EXECUTABLE=' + self.spec['bison'].prefix.bin.yacc,
+                '-DGIT_EXECUTABLE=' + self.spec['git'].prefix.bin.git,
+                '-DPCAP_INCLUDE_DIR=' + self.spec['libpcap'].prefix.include,
+                '-DPCAP_LIB=' + self.spec['libpcap'].prefix.lib + '/libpcap.so']
```

Comment:
Try `self.spec['libpcap'].libs`. This has the added benefit that it will also handle macOS where the file extension is `.dylib`.

Reply:
Thanks